nibabel/freesurfer/io.py

```diff
@@ -79,6 +79,16 @@ def read_geometry(filepath, read_metadata=False, read_stamp=False):
         Path to surface file
     read_metadata : bool
         Read metadata as key-value pairs
+        Valid keys:
+            'head' : array of int
+            'valid' : str
+            'filename' : str
+            'volume' : array of int, shape (3,)
+            'voxelsize' : array of float, shape (3,)
+            'xras' : array of float, shape (3,)
+            'yras' : array of float, shape (3,)
+            'zras' : array of float, shape (3,)
+            'cras' : array of float, shape (3,)
```

Comment:
I suspect this will look bad when rendered. Have you checked? This is more likely to work:
```
Valid keys:

    * ...
    * ...

```